docsrc/index.rst

```diff
@@ -54,6 +54,11 @@ User Guide
    tutorials/_rendered_examples/dynamo/vgg16_ptq
    tutorials/_rendered_examples/dynamo/engine_caching_example
    tutorials/_rendered_examples/dynamo/refit_engine_example
+   tutorials/serving_torch_tensorrt_with_triton
```

Comment:
I still want a tutorials section for stuff that is in beta or outside the mainline use cases